Makefile.dep

```diff
@@ -697,6 +697,11 @@ ifneq (,$(filter gcoap,$(USEMODULE)))
   USEMODULE += gnrc_sock_udp
 endif
 
+ifneq (,$(filter openthread,$(USEPKG)))
```

Comment:
I think the package dependencies could be added to a new `pkg/openthread/Makefile.dep` file instead. xtimer module could also be added as a dependency there.